.github/workflows/e2etest.yml

```
@@ -0,0 +1,83 @@
+          export REACT_APP_ENABLE_CREDENTIALS=$REACT_APP_ENABLE_CREDENTIALS >> .env
+          export REACT_APP_ENABLE_PASSWORDLESS=$REACT_APP_ENABLE_PASSWORDLESS >> .env
+          export REACT_APP_DEFAULT_AUTH_FLOW=$REACT_APP_DEFAULT_AUTH_FLOW >> .env
+          export REACT_APP_DISABLE_SIGNUP=$REACT_APP_DISABLE_SIGNUP >> .env
+          export REACT_APP_API_PATH=$REACT_APP_API_PATH >> .env
+          export REACT_APP_LD_CLIENT_ID=$REACT_APP_LD_CLIENT_ID >> .env
+          export REACT_APP_CONTENTFUL_SPACE_ID=$REACT_APP_CONTENTFUL_SPACE_ID >> .env
+          export REACT_APP_CONTENTFUL_DELIVERY_TOKEN=$REACT_APP_CONTENTFUL_DELIVERY_TOKEN >> .env
+          export REACT_APP_CONTENTFUL_COMPANY_DESCRIPTION_ID=$REACT_APP_CONTENTFUL_COMPANY_DESCRIPTION_ID >> .env
+          export REACT_APP_PRIMARY_THEME_COLOR=$REACT_APP_PRIMARY_THEME_COLOR >> .env
+          export REACT_APP_MEMBER_APP_HEADER_IMAGE=$REACT_APP_MEMBER_APP_HEADER_IMAGE >> .env
+          export REACT_APP_LOGO=$REACT_APP_LOGO >> .env
+          export REACT_APP_NAME=$REACT_APP_NAME >> .env
+          export REACT_APP_COMPANY_NAME=$REACT_APP_COMPANY_NAME >> .env
+          export REACT_APP_ENABLE_REQUIRE_SUBSCRIPTION=$REACT_APP_ENABLE_REQUIRE_SUBSCRIPTION >> .env
+          export REACT_APP_HIDE_NAV_BAR_HOME=$REACT_APP_HIDE_NAV_BAR_HOME >> .env
+          export REACT_APP_HIDE_NAV_BAR_CHAT=$REACT_APP_HIDE_NAV_BAR_CHAT >> .env
+          export REACT_APP_HIDE_NAV_BAR_APPOINTMENTS=$REACT_APP_HIDE_NAV_BAR_APPOINTMENTS >> .env
+          export REACT_APP_HIDE_NAV_BAR_PROFILE=$REACT_APP_HIDE_NAV_BAR_PROFILE >> .env
+          yarn playwright test
+          pwd
+          ls
+          cp -a /screenshots/. /playwright-report/
+      - uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: playwright-report
+          path: playwright-report/
+          retention-days: 30
```

This needs a notification to slack or somewhere, if not it will probably end up failing silently and could take a while before anybody notices

---

This is like any other part of CI that completes. To view the artifact this will be on the summary page of the gitub action as noted in the readme.

---

This is the same as if anyone else has a PR and the tests fail. We do not have slack alerting for failed PR CI tests in the frontend repos

---

We can leave it better than we found it. We should be notified of any impactful CI failures. If this breaks, we need to be alerted.

---

Good point.

---

As of right now this will run as part of every PR. There is a follow up ticket to limit it to manual trigger. I added the the notification.